extensions/redis-cache/runtime/src/main/java/io/quarkus/cache/redis/runtime/RedisCacheRuntimeConfig.java

```diff
@@ -5,55 +5,48 @@
 import java.util.OptionalInt;
 
 import io.quarkus.runtime.annotations.ConfigGroup;
-import io.quarkus.runtime.annotations.ConfigItem;
 
 @ConfigGroup
-public class RedisCacheRuntimeConfig {
+public interface RedisCacheRuntimeConfig {
     /**
      * The default time to live of the item stored in the cache.
      *
      * @deprecated Use {@link #expireAfterWrite} instead.
      */
-    @ConfigItem
-    @Deprecated
-    public Optional<Duration> ttl;
+    @Deprecated(forRemoval = true, since = "3.3.0")
+    Optional<Duration> ttl();
```

Comment:
@Ladicek I didn't want to do it in this PR but maybe we could drop this one?

Reply:
Agreed.